deploy/011-initial-liquidity.ts

```diff
@@ -10,7 +10,7 @@ import {
   getConfig,
   getTokenConfig,
 } from "../helpers/deploymentConfig";
-import { getUnderlyingMock, getUnderlyingToken, getUnregisteredVTokens } from "../helpers/deploymentUtils";
+import { getUnderlyingMock, getUnderlyingToken, getUnregisteredVTokens, toAddress } from "../helpers/deploymentUtils";
```

Comment:
Right all networks that have deployments saved should be marked as live and deployments should have a unique name so that we can fetch them from the external deployments defined in the hardhat config.